Source/WebCore/bindings/js/SerializedScriptValue.cpp

```diff
@@ -1390,6 +1433,51 @@ class CloneSerializer : CloneBase {
                 write(String::fromLatin1(JSC::Yarr::flagsString(regExp->regExp()->flags()).data()));
                 return true;
             }
+            if (auto* errorInstance = jsDynamicCast<ErrorInstance*>(obj)) {
+                auto& vm = m_lexicalGlobalObject->vm();
+                auto scope = DECLARE_THROW_SCOPE(vm);
+                auto errorTypeString = errorInstance->get(m_lexicalGlobalObject, vm.propertyNames->name).toWTFString(m_lexicalGlobalObject);
+                RETURN_IF_EXCEPTION(scope, false);
+
+                String message;
+                PropertyDescriptor messageDescriptor;
+                if (errorInstance->getOwnPropertyDescriptor(m_lexicalGlobalObject, vm.propertyNames->message, messageDescriptor) && messageDescriptor.isDataDescriptor())
+                    message = messageDescriptor.value().toWTFString(m_lexicalGlobalObject);
+                RETURN_IF_EXCEPTION(scope, false);
+
+                unsigned line = 0;
+                PropertyDescriptor lineDescriptor;
+                if (errorInstance->getOwnPropertyDescriptor(m_lexicalGlobalObject, vm.propertyNames->line, lineDescriptor) && lineDescriptor.isDataDescriptor())
+                    line = lineDescriptor.value().toNumber(m_lexicalGlobalObject);
```

Comment:
Ditto. `toNumber` requires error check. (Object can be returned, and it can have `valueOf` method).